parcel.go

```diff
@@ -36,18 +54,56 @@ func (s ParcelStore) GetByClient(client int) ([]Parcel, error) {
 	// заполните срез Parcel данными из таблицы
 	var res []Parcel
 
+	rows, err := s.db.Query("SELECT number, client, status, address, created_at FROM parcel WHERE client = :cl", sql.Named("cl", client))
+	if err != nil {
+		return res, err
+	}
+	defer rows.Close()
+
+	for rows.Next() {
+		p := Parcel{}
+
+		err := rows.Scan(&p.Number, &p.Client, &p.Status, &p.Address, &p.CreatedAt)
+		if err != nil {
+			return nil, err
+		}
+
+		res = append(res, p)
+	}
+
+	if err := rows.Err(); err != nil {
+		return res, err
```

Comment:
Лучше возвращать пустой ответ
```suggestion
		return nil, err
```